transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/Args.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.monster.encode.transformation
+
+import caseapp.{AppName, AppVersion, HelpMessage, ProgName}
+import org.broadinstitute.monster.EncodeTransformationBuildInfo
+
+@AppName("Encode transformation pipeline")
+@AppVersion(EncodeTransformationBuildInfo.version)
+@ProgName("org.broadinstitute.monster.etl.encode.EncodeTransformationPipeline")
+case class Args(
+  @HelpMessage("Path to the top-level directory where JSON was extracted")
+  inputPrefix: String,
+  @HelpMessage("Path where transformed JSON should be written")
+  outputPrefix: String
+)
```

Comment:
For the most part, this file was copied over from ClinVar. Maybe it would make sense to move it into a common utility at some point?